docs/core/build/emulator.md

```diff
@@ -52,6 +52,17 @@ brew install scons sdl2 sdl2_image pkg-config
 
 * __Windows__: not supported yet, sorry.
 
+## Rust
+
+You will require Rust and Cargo. The currently supported version is 1.52 stable.
+The recommended way to install both is with [`rustup`](https://rustup.rs/). If you
+are installing `rustup` for the first time, the stable toolchain will be installed
+for you automatically. Otherwise, make sure you are up to date:
+
+```sh
+rustup update
+```
+
```

Comment:
Perhaps also mention that clang/libclang is needed by [bindgen](https://rust-lang.github.io/rust-bindgen/requirements.html)?

Reply:
Fixed in a823ddf.